### DISCUSSION TITLE
Kibana 404 after Udpate to 2.3.110

### DISCUSSION THREAD

**Discussion by bauldini** [Security-Onion-Solutions/securityonion#7641] [Unsupported Versions] [answered]
Hello, 

I know there are some known issues with customer indexes with the upgrade to 2.3.110 but I haven't made any changes to the pillars to customize anything. Now after upgrading, I am getting a 404 on the Kibana page and the so-elatalert says missing when I run a so-status.  Am I missing a known issue? 

Thanks in advance.

**Comment by dougburks** [comment, marked as answer]
Were there any errors during the upgrade?

Have you checked `/opt/so/log/` for additional clues?

**Reply by bauldini** [reply]
There doesnt seem to be anything in regards to why elastalert is missing but I was able to get Kibana to come up after the update and now I am seeing the shards are failing. I am assuming there were errors during the upgrade as it ran over night.

**Reply by dougburks** [reply]
You should be able to review the soup log at `/root/soup.log`.

If you didn't find additional clues in `/opt/so/log/`, you might try something like:
```
sudo docker logs so-elastalert
```

**Reply by bauldini** [reply]
So when I run sudo docker logs so-elastalert  I get Error: No Such container: so-elastalert. I found a couple errors related to Kibana but nothing related to why the docker image would not be there. The soup log file actually shows it added the new container. When I do a docker image ls it actually shows the container.

**Reply by dougburks** [reply]
Have you tried restarting elastalert?
```
sudo so-elastalert-restart
```

Have you tried rebooting?

**Reply by bauldini** [reply]
Yes I have tried both.

**Reply by dougburks** [reply]
Have you made any changes to Elastalert configuration? If so, can you try reverting back to default configuration and then restarting Elastalert?

**Reply by bauldini** [reply]
You can close this. I found it to be more time effective to just rerun setup and start over. Thanks for your help.
